app/services/search_indexer.rb

```diff
@@ -31,7 +31,7 @@ def self.update_index(table, id, raw_data)
     end
 
     # for user login and name use "simple" lowercase stemmer
-    stemmer = table == "user" ? "simple" : Search.long_locale
+    stemmer = table == "user" ? "simple" : Search.ts_config
```

Comment:
PG took a text search configuration as the parameters for `TO_TSVECTOR` & `TO_TSQUERY`. It isn't related to locale so it's better for me to name with that intent.

From PG's doc:
```
CREATE TEXT SEARCH DICTIONARY english_ispell (
    TEMPLATE = ispell,
    DictFile = english,
    AffFile = english,
    StopWords = english
);
```